blocks/article-header/article-header.css

```diff
@@ -1,25 +1,30 @@
 /* this class styles the #main-content selector text within the article page */
 .article-content {
-  &
-    article
-    > :is(
-      div,
-      figure:not(.image-with-caption--full-bleed),
-      p,
-      h1,
-      h2,
-      h3,
-      h4,
-      h5,
-      h6,
-      ul,
-      ol,
-      dl,
-      aside
-    ):not(.article-header):not(.article-prefooter) {
+  & article > :is(
+    div,
+    figure:not(.image-with-caption--full-bleed),
+    p,
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6,
+    ul,
+    ol,
+    dl,
+    aside
+  ):not(.article-header, .article-prefooter) {
```

Comment:
Combined the two nots into one and just adjusted indentation here for readability.